cwltool/main.py

```diff
@@ -695,6 +697,18 @@ def main(argsl=None,  # type: List[str]
                 argsl = sys.argv[1:]
             args = arg_parser().parse_args(argsl)
 
+        # If On windows platform, A default Docker Container is Used if not explicitely provided by user
+        if onWindows() and not args.default_container:
+            args.default_container = "ubuntu"
```

Comment:
Ok. One problem that we still have is that test 52 and 53 uses `rev` command. It is not posix compliant so ubuntu and alpine have different implmentation thus it gives wrong output. I thinking of adding a ubuntu container for these tests.